### ISSUE TITLE
zoom in yaxis

### ISSUE THREAD

**Issue by Sumon-miazi** [apexcharts/apexcharts.js#794]
Is there any way to zoom line chart in the y-axis. if I select some area(blue color rectangle) I want that area to zoom in the y-axis(black arrow). Is it possible?
![DeepinScreenshot_select-area_20190724160244](https://user-images.githubusercontent.com/28239287/61785038-b2055a00-ae2c-11e9-8434-7d076107be2d.png)

I wrote this in my chart.
`chart: {
                id: 'chart2',
                type: 'line',
                height: 400,
                animations: {
                  enabled: false,
                },
                zoom: {
                  enabled: true,
                  type: 'xy',
                  autoScaleYaxis: true,  
                  zoomedArea: {
                    fill: {
                      color: '#90CAF9',
                      opacity: 0.4
                    },
                    stroke: {
                      color: '#0D47A1',
                      opacity: 0.4,
                      width: 1
                    }
                  },
                },
              },`

**Comment by junedchhipa** [comment]
If you don't want to zoom in `x` along with `y`, you can do

```javascript
zoom: {
  type: 'y'
}
```

Let me know if it worked.

**Comment by junedchhipa** [comment]
Just checked it again, looks like the functionality is broken.
I will fix and update this thread.

**Comment by Sumon-miazi** [comment]
I tried that. but didn't work.

**Comment by Sumon-miazi** [comment]
> Just checked it again, looks like the functionality is broken.
> I will fix and update this thread.

**Comment by Sumon-miazi** [comment]
> Just checked it again, looks like the functionality is broken.
> I will fix and update this thread.

thanks

**Comment by junedchhipa** [comment]
Fixed it.
The patch will be released in 3.8.4

**Comment by Sumon-miazi** [comment]
zoom in y and xy axis is not working in 3.8.4v. @junedchhipa 

```
zoom: {
        enabled: true,
        //type:'xy',
        type: 'y',  
},
```

![deepin-screen-recorder_Select area_20190806172733](https://user-images.githubusercontent.com/28239287/62536632-3f569e80-b870-11e9-9df5-5a2449329616.gif)
